### PR TITLE
Modified README

### LINKED ISSUE #48: Update README.md
Change site link from surge to netlify deployment.

@saranshbarua We need to update repo description for the same too.

### PR DESCRIPTION
**Pull request by sidntrivedi**
Fixes #48 .

PTAL @saranshbarua. Thanks.

### CODE REVIEW

**Comment by ryzokuken** [comment]
@sidntrivedi012 "Solves" is neither the correct terminology nor is it detected by GitHub. Please use "Fixes" moving forwards.

**Comment by sidntrivedi** [comment]
@ryzokuken Did it by mistake. Made the changes. Thanks.

**Comment by saranshbarua** [comment]
@sidntrivedi012 squash your commits because that helps in keeping the history clean.

**Review by saranshbarua — CHANGES_REQUESTED** [review]
You might want to delete the binary file because that's unnecessary

**Comment by sidntrivedi** [comment]
@saranshbarua PTAL. Squashed the commits into one and deleted the binary.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
+<p align="center"><img src="osdc.gif"></p>
 
 ## Setup
-`yarn`
 
-`npm install --global gatsby-cli`
+- Fork and then clone the repository.
+- Install [yarn](https://yarnpkg.com/lang/en/docs/install/#mac-stable)
+- Run `yarn` to install the necessary packages.
+- Run `gatsby develop` to run the website in development mode on `localhost:8000`.
```
Comment (saranshbarua) [inline README.md:17]: Need to install gatsby-cli before `gatsby develop`, maybe add that too

**Review by saranshbarua — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (saranshbarua) [reply README.md:17]: Apart from this everything else looks good. Please make the changes and we'll be good to go.

Comment (sidntrivedi) [reply README.md:17]: Sure. Making the changes. I though gatsby-cli would be one of the packages installed by yarn. But it was just gatbsy.

Comment (sidntrivedi) [reply README.md:17]: @saranshbarua PTAL. Done.

Comment (saranshbarua) [reply README.md:17]: Great. LGTM.

**Pull request merged**